### ISSUE TITLE
Text in searchbar and urlbar cannot be interacred with mouse

### ISSUE THREAD

**Issue by Naezr** [Tagggar/Firefox-Alpha#4]
![изображение](https://github.com/Tagggar/Firefox-Alpha/assets/95460152/b9ff1155-2852-47a4-968e-6b78a61b19d3)
![изображение](https://github.com/Tagggar/Firefox-Alpha/assets/95460152/1bef0875-f77e-424a-94ab-b21e4f0077d4)
This text cannot be selected with the mouse

**Comment by Tagggar** [comment]
This is because the interaction is not direct with the navbar, but with the invisible capture-box. We cannot make changes to the browser's source code; our options are limited to modifying the visuals using CSS exclusively.

I know, the current solution is `🩼not ideal🩼`, but it works without third-party scripts with minimal browser modification.
A small price to pay for the convenience of familiar interactions as on mobile devices.

In future releases, perhaps I'll come up with ideas on how to incorporate more interactions into the **[Combined Tab and Urlbar](https://github.com/Tagggar/Firefox-Alpha#combined-tab-and-urlbar)**.
Nevertheless, the text in the `URL bar` can still be edited via the keyboard.

**Comment by Naezr** [comment]
Okay, I get it. But I found another problem that makes it impossible to edit text in the address bar even with the keyboard. It is the lack of scrolling when the text goes out of view. Is it possible to add the same scrolling as, for example, in the search field?
![firefox_T1TgoELga9](https://github.com/Tagggar/Firefox-Alpha/assets/95460152/67b844a1-4fd9-4c44-bf62-e7acc011bd7f)
![firefox_kDRfPxYhhv](https://github.com/Tagggar/Firefox-Alpha/assets/95460152/47047dd7-a541-40b7-8840-88b041c35c06)

**Comment by Naezr** [comment]
I also found a thread on reddit where someone was able to move the actual urlbar to a tab using javascript. I think this might help you in creating a more usable theme.
https://www.reddit.com/r/FirefoxCSS/comments/nv1lb6/url_on_the_tab_like_the_new_safari/
![изображение](https://github.com/Tagggar/Firefox-Alpha/assets/95460152/dc9bda87-d849-404f-ac0e-906d117e7faf)

**Comment by Tagggar** [comment]
Interesting, I'll take a look.

Configuring the position of the URL bar using JavaScript can be quite intricate, given that the tabs lack a position variable, requiring extensive debugging.

Currently, I believe that a native-like  `tab + urlbar` solution is possible by fundamentally changing the code for both the `urlbar` and `tabstoolbar` components.

**Comment by Tagggar** [comment]
Certainly, I have some concepts for a CSS-only implementation. Further testing is required.

If the outcome meets my expectations, I plan to release them following the Firefox 120 update.
